src/routes/expenses.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const express = require('express');
+
+const expenseControllers = require('../controllers/expense');
+
+const router = express.Router();
+
+router.get('/expenses', express.json(), expenseControllers.getAllExpenses);
```

Comment:
I think it's better to add app.use(express.json()) in createServer to avoid repeating of code

Reply:
agree, move express.json() to server